modules/op-tkn-pipelines-compatibility-support-matrix.adoc

```diff
@@ -13,22 +13,20 @@ In the table, features are marked with the following statuses:
 TP:: Technology Preview
 GA:: General Availability
 
-// Writer, see http://dashboard.apps.cicd.ospqa.com/releases/componentmatrix/
-
 .Compatibility and support matrix
 [options="header"]
 |===
 
-| {pipelines-title} Version 8+| Component Version | OpenShift Version | Support Status
+| {pipelines-title} Version 10+| Component Version | OpenShift Version | Support Status
 
-| Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | |
+| Operator | Pipelines | Triggers | CLI | Chains | Hub | {pac} | Results | Manual Approval Gate | Pruner | Cache | |
 
 //TODO UPDATE
-|1.20 | 1.3.x | 0.33.x | 0.42.x | 0.25.x (GA) | 1.22.x (TP) | 0.37.x (GA) | 0.16.x (GA) | 0.6.x (TP) | 4.15, 4.16, 4.17, 4.18, 4.19 | GA
+|1.21 | 1.6.x | 0.34.x | 0.43.x | 0.26.x (GA) | 1.23.x (TP) | 0.39.x (GA) | 0.17.x (GA) | 0.7.x (TP) | 0.3.x (GA) |0.3.x (GA) | 4.16, 4.17, 4.18, 4.19, 4.20 | GA
```

Comment:
For exact component versions, can you please take the reference from here https://spaces.redhat.com/spaces/pipelines/pages/716243269/1.21+Components+Release+Dates+Stats